update_employees.py

```diff
@@ -137,22 +147,19 @@ def get_employee_data():
     json_raw = wddx.loads(res.text)
     #  remove weird leading slashes from data contents
     json_clean = json_raw[0].replace("//", "")
-    records_hr_unfiltered = json.loads(json_clean)
-    return drop_empty_positions(records_hr_unfiltered)
+    return json.loads(json_clean)
 
 
-def create_placeholder_email(record, name_field):
+def create_placeholder_email():
     """
-    Emails are required for Knack records. We can guess a temporary email for a user so they can be added to knack
-    :param record: employee record
-    :param name_field: name field in knack
-    :return: temporary placeholder email
+    Emails are required for Knack user records. When an employee doesn't have an email
+    address, we generate a unique address using the current timestamp
```

Comment:
by using a random email address we are avoiding future email address collisions for when we try to generate the email address for a non-email-address-holding  employee who is being re-hired. E.g. a seasonal employee.

the assumption here is that if the user does not have an email address then they do not need to login to the HR portal. if HR wants such an employee to be able to access the portal, they need to work with CTM to get the user a valid email address and to update that info the upstream system serving this API.